### PR TITLE
Pass chromium source context to devtools e2e tests

### PR DESCRIPTION
**Pull request by ryanjduffy**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -211,6 +211,14 @@ steps:
     trigger: "runtime-fe-end-to-end-tests"
     depends_on:
       - "build-chromium-linux-x86_64"
+    build:
+      env:
+        REPLAY_API_KEY: "rwk_o4eIfrkch6Y7eKFbJPGtKrVxNh0UiTF81KYU4IE3otr"
```
Comment (Domiii) [inline .buildkite/pipeline.yml:216]: Accidentally exposed `API_KEY`

Comment (ryanjduffy) [reply .buildkite/pipeline.yml:216]: Not really, just a test API key to see if this will work. But point taken this should be in a secret

**Comment by Domiii** [comment]
I am not sure how to review this. Will tag in others who might be more suitable here.
Also: Can we test this before landing?

**Comment by ryanjduffy** [comment]
Once the [devtools PR](https://github.com/replayio/devtools/pull/10237) lands, we can retrigger this CI workflow on this PR and it should comment here.

**Review by jazzdan — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -211,6 +211,15 @@ steps:
     trigger: "runtime-fe-end-to-end-tests"
     depends_on:
       - "build-chromium-linux-x86_64"
+    build:
+      env:
+        RUNTIME_TEAM_API_KEY: "rwk_o4eIfrkch6Y7eKFbJPGtKrVxNh0UiTF81KYU4IE3otr"
```
Comment (toshok) [inline .buildkite/pipeline.yml:216]: this feels like a bad idea.  can it come from secrets instead?

Comment (ryanjduffy) [reply .buildkite/pipeline.yml:216]: Yeah ... see Domi's comment above

Comment (toshok) [reply .buildkite/pipeline.yml:216]: odd.  Thought I'd looked at conversation before commenting.

**Review by toshok — APPROVED** [review]

**Pull request merged**